dist.ini

```diff
@@ -1,69 +1,16 @@
 name    = Data-Validate-Sanctions
 author  = binary.com <BINARY@cpan.org>
 license = Perl_5
-copyright_holder = binary.com
-copyright_year   = 2017
+copyright_holder = Binary Services Ltd
+copyright_year   = 2022
 
-[Git::GatherDir]
-exclude_filename = Makefile.PL
-include_dotfiles = 1
-[PruneCruft]
-except = t/rc/\.perl.*rc$
-[ManifestSkip]
-[MetaYAML]
-[License]
-[Readme]
-[MakeMaker]
-eumm_version = 6.48
-prereq_fatal = 1
-[ExecDir]
-[ShareDir]
-dir = share
-[Manifest]
-[TestRelease]
-[ConfirmRelease]
-[Prereqs::FromCPANfile]
-[Prereqs / BuildRequires]
-perl = 5.014000
-[CheckPrereqsIndexed]
-[CheckExtraTests]
-[VersionFromModule]
-[ReversionOnRelease]
-[NextRelease]
-[OurPkgVersion]
-[GitHub::Meta]
-repo = binary-com/perl-Data-Validate-Sanctions
-[InstallGuide]
-[MetaJSON]
-[InsertExample]
+[@Author::DERIV]
+allow_dirty = lib/Data/Validate/Sanctions.pm
+-remove = PodInherit
+[Encoding]
+encoding = bytes
+match    = ^t/data/.*\.zip$
```

Comment:
Just a bit curious. What does this section do ?

Reply:
https://metacpan.org/pod/Dist::Zilla::Plugin::Encoding
https://metacpan.org/pod/Dist::Zilla::Plugin::AutoPrereqs
```
Note, if you have any non-Perl files in your t/ directory or other directories being scanned, be sure to mark those files' encoding as bytes with the [Encoding](https://metacpan.org/pod/Dist::Zilla::Plugin::Encoding) plugin so they won't be scanned:
```